docs/intelligence_center/integrations.md

```diff
@@ -0,0 +1,29 @@
+# Intelligence Center external integrations
+
+The intelligence center data can be consummed using several third party integrations.
+
+## MISP Feed
+
+The default feed is available as a MISP feed.
+
+It can be added to an existing MISP instance by following [MISP's documentation](https://www.circl.lu/doc/misp/managing-feeds/).
+
+The following field values are required for the feed to work properly:
+
+-   Input Source: Network
+-   URL: `https://api.sekoia.io/api/v2/inthreat/misp`
+-   Source Format: MISP Feed
+-   Headers: `Authorization: Bearer <APIKEY>`
```

Comment:
Shouldn’t we also explain how to manage APIKeys? BTW, it seems that we don’t have such a page currently.

Reply:
Yes I agree that it would be useful. We should create a page and add a link to it in this section.

Reply:
I planned to add pages concerning all the community/user management part soon, but if someone wants to do it, fell free !